examples/apply-setters/simple/README.md

```diff
@@ -48,17 +52,18 @@ Invoking `apply-setters` function would apply the changes to resource configs
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: my-new-map # kpt-set: ${name}
+  name: darwin-development # kpt-set: ${image}-development
 data:
   some-key: some-value
 ---
 apiVersion: v1
 kind: MyKind
 metadata:
-  name: ns
-environments: # kpt-set: ${env}
-  - prod
-  - stage
+  name: darwin # kpt-set: ${image}
+image: nginx:2.1.2 # kpt-set: ${app}:${tag}
+roles: # kpt-set: ${role}
+  - dev
+  - intermediate
 ```
 
 ### Function invocation
```

Comment:
> $ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/apply-setters/simple .

No `.` needed here, right?
Can you please update other examples as well?

Reply:
Done.